File: official/docs/c#/insurances/RetrieveInsuranceList.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

Dictionary<string, object> listParams = new Dictionary<string, object>() {
  { "page_size", 5 }
};

InsuranceCollection insuranceCollection = await Insurance.All(listParams);
InsuranceCollection nextInsuranceCollection = await insuranceCollection.Next();

File: official/docs/node/insurances/retrieve_insurance_list.js
const Easypost = require('@easypost/api');
const api = new Easypost('EASYPOST_API_KEY');

api.Insurance.all({
    page_size: 5,
    start_datetime: '2016-01-02T08:50:00Z',
  }).then(console.log);
